docs/design/abstractions/project-structure.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: "Mage file directory"
+description: "Details about how the Mage file directory works and how typical projects are structured."
+---
+
+## Overview
+
+A Mage workspace lives inside it's own folder. It's there where mage data is stored (`mage_data`) and individual projects are housed. You can think of a project as an environment: it contains configurations, pipelines, and other project-specific data. A workspace can hold multiple projects.
```

Comment:
The definition might not be proper. It's conflicted with the workspace definition in this doc: https://docs.mage.ai/production/developing-in-the-cloud/workspaces/overview